x-pack/plugins/monitoring/public/components/no_data/reasons/__tests__/__snapshots__/we_tried.test.js.snap

```diff
@@ -28,7 +28,7 @@ Array [
       No Monitoring data could be found for the selected time period, but we could not find the cluster setting that makes the data unavailable.
     </p>
     <p>
-      There may be data available for a different time period than we have selected. Try adjusting the time filter controls to a time range where the Monitoring data is expected.
+      There may be data available for a different time period than we have selected. Make sure the time filter is set to "Last 1 hour."
     </p>
     <p>
       We are refreshing the search for data in the background. If cluster data is found, we will redirect to the cluster overview page.
```

Comment:
The first sentence is a little hard to parse.  Is the second half necessary?  The CTA is in the second paragraph, and it would be nice to get that information to the user sooner.  So how about shortening it a bit.

-----

We couldn't activate monitoring

No monitoring data found.  Try setting the time filter to "Last 1 hour" to see if data is available for a different time period.

If data is in your cluster, you'll see your monitoring dashboard here.